app/lib/services/voip/useCallStore.test.ts

```
@@ -1,6 +1,34 @@
+		getLastVoipToken: jest.fn(() => ''),
+		stopNativeDDPClient: jest.fn(),
+		stopVoipCallService: jest.fn(),
+		addListener: jest.fn(),
+		removeListeners: jest.fn()
+	}
+}));
+
 import type { IClientMediaCall } from '@rocket.chat/media-signaling';
 
 import { useCallStore } from './useCallStore';
+import { voipNative, InMemoryVoipNative } from './VoipNative';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)